index.html

```diff
@@ -89,27 +141,94 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    state: 'all'
   };
 
-  var svg;
+  var svg, x , y, histScale;
+  var margin = { top: 15, right: 15, bottom: 40, left: 60 }
+  var width = 840 - margin.left - margin.right;
+  var height = 600 - margin.top - margin.bottom;
 
   // FETCH DATA
 
+  d3.json('party_contr_by_zip.json', function(error, json) {
+    if (error) { throw error; }
+
+    data = json;
+
+    setup();
+    update();
+  });
+
   function setup() {
     // SVG
 
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr("width", width + margin.right + margin.left)
+      .attr("height", height + margin.top + margin.bottom)
+      //.style("background", "lightgrey")
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
     // SCALES
 
+    x = d3.scale.linear()
+          .domain([0, d3.max(data, function (d) { return d.democratContribution; })])
+          .range([0, width])
+          .nice();
+
+    y = d3.scale.linear()
+          .domain([0, d3.max(data, function (d) { return d.republicanContribution; })])
+          .range([height, 0])
+          .nice();
+
```

Comment:
Given the high degree to which your data is clustered within the lower range of this data (0-3 million) this is a really strong case for a log scale (based 10 maybe). This would spread out your data so you actually see more of it.

Alternatively I might suggest cutting out some of the outliers.